pootle/apps/pootle_app/management/commands/initdb.py

```diff
@@ -33,7 +33,7 @@ class Command(NoArgsCommand):
 
     def handle_noargs(self, **options):
         self.stdout.write('Populating the database.')
-        initdb(options["create_projects"])
+        InitDB().init(options["create_projects"])
```

Comment:
I thought that this would be changed to pass the options to `__init__` instead of `initdb`.